webpack/assets/javascripts/foreman_tools.js

```diff
@@ -55,10 +57,6 @@ export function activateTooltips(elParam = 'body') {
       return this.scrollWidth > this.clientWidth ? this.textContent : null;
     },
   });
-  el
-    .find('*[title]')
```

Comment:
i believe this will break other consumers of this?

Reply:
yes, i'm readding this and changing the `not()`

`.not('*[rel],.fa,.pficon')`